fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/FDBLuceneQueryTest.java

```diff
@@ -1399,4 +1405,35 @@ void testQueryWithManyDocuments() {
             assertPrimaryKeys("text:morningstart", false, Set.of());
         }
     }
+
+    @ParameterizedTest
+    @RandomSeedSource
+    void encrypted(long seed) throws Exception {
```

Comment:
Perhaps worth having some of the tests in LuceneIndexMaintenanceTest chose to randomly encrypt and-or compress.

Reply:
I added this as an argument to the existing tests that took a bunch of arguments.